### PR TITLE
Fixed App Intro Bug

### LINKED ISSUE #1199: Bug while starting the application for first time
**Actual Behaviour**
Currently when the user starts the app , a dialog box appears which asks for granting permission to access photos and media files in the mobile.If we select **DENY** then we can still see the 3 introduction statements on the splash screen.For clarity see the attached GIF below. 

**Steps to reproduce it**
1)Open the app.
2) When the app asks for accessing media files permission , select **DENY**.

**Screenshots of the issue**
giphy 1

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fix #1199 

Changes: Now when the user denies the permission to access external storage the Showcase Sequence doesn't begin.

### CODE REVIEW

**Comment by sridharjajoo** [comment]
@anantprsd5 @mohitmanuja kindly review.

File: app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java [hunk 1]
```diff
@@ -18,6 +19,7 @@
 import org.fossasia.phimpme.gallery.activities.LFMainActivity;
 import org.fossasia.phimpme.opencamera.Camera.CameraActivity;
 
+
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java]: @sridharjajoo Avoid unnecessary new lines

Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java]: @sridharjajoo Avoid unnecessary new lines

File: app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java [hunk 2]
```diff
@@ -55,7 +57,10 @@ protected void onCreate(Bundle savedInstanceState) {
          nav_cam = (BottomNavigationItemView) findViewById(R.id.navigation_camera);
          nav_acc = (BottomNavigationItemView) findViewById(R.id.navigation_accounts);
 
-        presentShowcaseSequence(); // one second delay
+        int checkStoragePermission = ContextCompat.checkSelfPermission(this, android.Manifest.permission.WRITE_EXTERNAL_STORAGE);
+
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java]: @sridharjajoo Here also.

**Comment by sridharjajoo** [comment]
@anantprsd5 Done the changes.

**Pull request merged**